TrenDAP/wwwroot/TypeScript/Features/DataSets/DataSet.tsx

```diff
@@ -68,37 +68,20 @@ const DataSet: React.FunctionComponent<IProps> = (props: IProps) => {
    
     return (
         <>
-            <ul className="nav nav-tabs" style={{padding:10}}>
-                <li className="nav-item">
-                    <a className="nav-link active" data-toggle="tab" onClick={() => setTab('settings') }>Settings</a>
-                </li>
-                {
-                    props.Connections.map((conn, index) => (
-                        <li className={"nav-item " + styles.workspacetab}  key={index}>
-                            <a className="nav-link" data-toggle="tab" onClick={() => setTab(index.toString())}>
-                                {dataSources.find(ds => ds.ID === conn.DataSourceID)?.Name}
-                            </a>
-                            <span onClick={() => deleteConn(index)}>X</span>
-                        </li>
-                    ))
-                }
-            </ul>
-            <div className="tab-content">
-                <div className={"tab-pane container " + (tab === "settings" ? 'active' : 'fade')}>
+        <div className="tab-content" style={{height: '100%', width: '100%'}}>
+                <div className={"tab-pane container " + (props.Tab === "settings" ? 'active' : 'fade')} style={{height: '100%', width: '100%'}}>
                     <DataSetGlobalSettings DataSet={props.DataSet} SetDataSet={props.SetDataSet} Connections={props.Connections} SetConnections={props.SetConnections} />
                 </div>
                 {
                     props.Connections.map((conn, index) => (
-                        <div className={"tab-pane container " + (tab === index.toString() ? 'active' : 'fade')} id={index.toString()} key={index}>
+                        <div className={"tab-pane container " + (dataSources.find(ds => ds.ID === conn.DataSourceID)?.Name + index.toString() === props.Tab ? 'active' : 'fade')} id={index.toString()} key={index}>
```

Comment:
are we fxing these later? `fade` class causes issues with rendering anyway/ not calling `useEffects` when selected

Reply:
Yes, that gets fixed up the waterfall.

Reply:
It looks like this file got reworked by Gabriel at some point, as in `TDAP-6` this file doesn't exist, and after doing a global search for the `tab-pane container` class and I don't see anywhere that we are using the fade class.